app/Modules/ExpertPanel/Events/ApplicationInitiated.php

```diff
@@ -37,5 +39,17 @@ public function getStep()
     {
         return 1;
     }
+
```

Comment:
fromActivity "reconstitutes" the event from an activity log.  Events can then be "published" to a `StreamMessage`.